src/olympia/discovery/serializers.py

```diff
@@ -58,3 +58,7 @@ def to_representation(self, instance):
             data['description'] = (
                 u'<blockquote>%s</blockquote>' % instance.addon.summary)
         return data
+
+
+class DiscoveryRecommendationSerializer(DiscoverySerializer):
+    is_recommendation = serializers.BooleanField()
```

Comment:
Do you think it's worth adding a new serialized compared to just adding the field to `DiscoverySerializer` ? the source will have it anyway...

Reply:
https://github.com/mozilla/addons-server/pull/6410/commits/82a48de885b5330992caebf435feb58b87c1f7f9